app/presenters/publishing_api/historical_accounts_index_presenter.rb

```diff
@@ -0,0 +1,69 @@
+module PublishingApi
+  class HistoricalAccountsIndexPresenter
+    attr_accessor :update_type
+
+    def initialize(update_type: nil)
+      self.update_type = update_type || "major"
+    end
+
+    def content_id
+      "a258e45a-acbe-4d70-ad2c-a2a20761536a"
+    end
+
+    def content
+      content = BaseItemPresenter.new(
+        nil,
+        title: "Past Prime Ministers",
+        update_type:,
+      ).base_attributes
+
+      content.merge!(
+        base_path:,
+        details: {
+          appointments_without_historical_accounts:,
+        },
+        document_type: "historic_appointments",
+        public_updated_at: Time.zone.now,
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "historic_appointments",
+      )
+
+      content.merge!(PayloadBuilder::Routes.for(base_path))
+    end
+
+    def find_appointments_without_accounts(role)
+      past_pms = role.role_appointments
+      ids_of_pms_without_accounts = past_pms.map(&:person_id) - HistoricalAccount.all.map(&:person_id)
+      Person.where(id: ids_of_pms_without_accounts)
```

Comment:
Rails can do the association stuff for you, without needing to convert IDs to people:
```suggestion
      role.role_appointments.map(&:person) - HistoricalAccount.all.map(&:person)
```

If you change this, it might be worth just inlining it on lines 41-42 instead of being a method.